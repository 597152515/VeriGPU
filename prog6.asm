out abx
out cdx
out 12x
out 34x
li x1 01x
li x2 23x
outr x1
outr x2
li x2 56x
li x1 78x
outr x1
outr x2

20x:
short deadx
short beefx
