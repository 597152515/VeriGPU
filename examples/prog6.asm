# not sure why I wrote this :P
# seems identical to prog5?

li x1 7
li x2 35
outr x1
outr x2
li x2 22
li x1 47
outr x1
outr x2

outr x1
outr x2

halt
