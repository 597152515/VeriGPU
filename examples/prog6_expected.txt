out 0 00000000000000000000000000011011 0000001b 27
out 1 00000000000000000000000000101101 0000002d 45
out 2 00000000000000000000000000110010 00000032 50
out 3 00000000000000000000000001000100 00000044 68
out 4 00000000000000000000000000000001 00000001 1
out 5 00000000000000000000000000100011 00000023 35
out 6 00000000000000000000000001111000 00000078 120
out 7 00000000000000000000000001010110 00000056 86